docs/tools/dependency-track.md

```diff
@@ -0,0 +1,126 @@
+# Dependency Track
+Depencency Track is a tool that allows centralised monitoring of vulnerabilities imported through the dependencies of your applications. This tool bases his security analysis solely on a provided SBOM.
+
+An SBOM (Software Bill of Materials) is a "list of ingrediënts" that make up your software. This list is compared to a set of CVE reports to analyse the included vulnerabilities and their severity.
+
+## Why would you need an SBOM?
+Supply chain attacks are in the rise. It is important that component providers also provide this format to ensure a certain level of security. The American government requires software suppliers to include an SBOM to ensure the safety and integrity of the components they are providing.
+
+## Setting up Dependency-Track
+The application itself has a straightforward and well-documented setup:
+- Deployment through WAR (deprecated): https://docs.dependencytrack.org/getting-started/deploy-exewar/
+- Deployment through a docker container: https://docs.dependencytrack.org/getting-started/deploy-docker/
+
+The application will run 2 separate docker containers by default, frontend and backend separately. It's recommended that you also provide the application with its dedicated database.
+
+I've set up a compose script that will help you start out. It contains the frontend, backend and dedicated postgres DB: 
+
+````
+version: '3.7'
+
+#####################################################
+# This Docker Compose file contains three services
+#    Dependency-Track API Server
+#    Dependency-Track FrontEnd
+#    Postgres Database
+#####################################################
+
+volumes:
+  dependency-track:
+  postgres:
+services:
+  dtrack-apiserver:
+    image: dependencytrack/apiserver
+    depends_on:
+      - postgres
+    environment:
+      # The Dependency-Track container can be configured using any of the
+      # available configuration properties defined in:
+      # https://docs.dependencytrack.org/getting-started/configuration/
+      # All properties are upper case with periods replaced by underscores.
+      #
+      # Database Properties
+      - ALPINE_DATABASE_MODE=external
+      - ALPINE_DATABASE_URL=jdbc:postgresql://10.3.61.1:5433/dtrack
+      - ALPINE_DATABASE_DRIVER=org.postgresql.Driver
+      - ALPINE_DATABASE_USERNAME=dtrack
+      - ALPINE_DATABASE_PASSWORD=admin
+      - ALPINE_DATABASE_POOL_ENABLED=true
+      - ALPINE_DATABASE_POOL_MAX_SIZE=20
+      - ALPINE_DATABASE_POOL_MIN_IDLE=10
+      - ALPINE_DATABASE_POOL_IDLE_TIMEOUT=300000
+      - ALPINE_DATABASE_POOL_MAX_LIFETIME=600000
+    deploy:
+      resources:
+        limits:
+          memory: 12288m
```

Comment:
Are these the recommended memory requests and limits? 
Damn those are even higher than I remembered.